### ISSUE TITLE
OS line displays wrong OS arch for Apple Silicon.

### ISSUE THREAD

**Issue by chrmatique** [aleksilassila/uebersicht-fetch#4]
I have an M1 MacBook Air and I noticed that the OS displays the wrong architecture for Apple Silicon. Now I've found this is very odd as in `runOnce.sh` it collects the architecture via calling `$(uname -m)` now when I run this in my terminal it does return the proper architecture:
```
╭─cr@cr-mba ~/git-repos/uebersicht-fetch ‹main› 
╰─$ uname -m
arm64
```
But when it runs inside Ubersicht it displays `x86_64`(Intel) and not `arm64`(Apple Silicon). I currently haven't found a fix for this, I'm currently working on a fix and will update this issue when I make some progress.

![Screen Shot 2021-07-19 at 22 11 15](https://user-images.githubusercontent.com/5068526/126271162-da091216-5f54-42b7-bbac-07e47113da64.jpg)

**Comment by aleksilassila** [comment]
I don't have M1 myself so I can't test this, but if you figure it out let me know

**Comment by chrmatique** [comment]
@aleksilassila my latest commit to #3 fixes this issue. 😄
It appears that Übersicht runs on top of Rosetta 2 and `uname -m` doesn't detect that properly so you have to dig into `sysctl.proc_translated` to get the status of if the current process is running through Rosetta. Little weird but it works fine now.

**Comment by aleksilassila** [comment]
Merged #3
